### PR TITLE
fix: add BoseSj

### LINKED ISSUE #1638: [PROFILE ADDITION] <SJ Basak>
### Profile screenshot

screencapture-github-BoseSj-2023-01-10-16_27_40

### PR DESCRIPTION
**Pull request by BoseSj**
resolves #1638

resolves #issueNumber

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/BoseSj.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/58129377/211534605-44b02824-5f70-43eb-bd3f-92ca72c85a0f.png",
```
Comment (krshkun) [inline profiles/BoseSj.json:2]: I suggest more of a image like this:
image


```suggestion
  "image": "https://user-images.githubusercontent.com/91655303/211567625-1e497baa-9b02-4376-a688-ac000ac894d5.png",
```

Comment (BoseSj) [reply profiles/BoseSj.json:2]: Okay i see, you mean the animated "Contribution Graph" should be visible like that? Right?

Comment (BoseSj) [reply profiles/BoseSj.json]: Okay I made the changes, please review it.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**